course-generator/cp.lua

```diff
@@ -230,7 +230,6 @@ function courseGenerator.generate( vehicle )
 		CourseplayEvent.sendEvent(vehicle, "self.cp.courseWorkWidth", vehicle.cp.courseWorkWidth) -- need a setting for this one
 		CourseplayEvent.sendEvent(vehicle, "self.cp.workWidth", vehicle.cp.workWidth) -- need a setting for this one
 		CourseplayEvent.sendEvent(vehicle, "self.cp.laneNumber", vehicle.cp.laneNumber) -- need a setting for this one
-		CourseplayEvent.sendEvent(vehicle, "self.cp.toolOffsetX", vehicle.cp.toolOffsetX) -- need a setting for this one
 		CourseplayEvent.sendEvent(vehicle, "self.cp.toolOffsetZ", vehicle.cp.toolOffsetZ) -- need a setting for this one
```

Comment:
@schwiti6190 not sure if this is ok, since toolOffsetX now derived from Setting, what else needs to be done to work in MP?